numba/core/ir_lexer.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this necessary in python 3?

Reply:
In case our lexer will be included in Pygments  official repo.

Reply:
But you are right, not necessary